src/makeCalculator.js

```diff
@@ -4,7 +4,47 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    // Метод для додавання
+    add: function (a) {
+      this.result += a;
+    },
+
+    // Метод для віднімання
+    subtract: function (a) {
+      this.result -= a;
+    },
+
+    // Метод для множення
+    multiply: function (a) {
+      this.result *= a;
+    },
+
+    // Метод для ділення
+    divide: function (a) {
+      if (a !== 0) {
+        this.result /= a;
+      } else {
+        return 'Cannot divide by zero!';
+      }
+    },
+
+    // Метод для скидання результату
+    reset: function () {
```

Comment:
[CODE KNOWLEDGE] - Use the method shorthand syntax for the `reset` method as well. Change `reset: function () {` to `reset() {`.